pywren/ibmcf/default_config.yaml.template

```diff
@@ -8,18 +8,19 @@ ibm_cf:
     api_key     : <CF_API_KEY>
     action_name : pywren_3.6  # Default runtime
 
-storage_backend: ibm_cos
+#internal_storage_backend: ibm_cos
+#function_storage_backend: ibm_cos
 
```

Comment:
@JosepSampe what do you mean by function_storage_backend?

Reply:
@JosepSampe we need better naming ;) may be simply call is 'storage' ?

Reply:
@JosepSampe may be even simpler...to call it "cos", but this might be problematic if we plan to support for storage as an input

Reply:
@gilv **function_storage_backend** is only a config variable name that determines wich storage client will get a user that writes **storage_handler** (or whatever else name we want) in the parameters of the function.